src/Layers/xrRenderGL/glTexture.cpp

```diff
@@ -121,21 +121,31 @@ GLuint CRender::texture_load(LPCSTR fRName, u32& ret_msize, GLenum& ret_desc)
         glTexParameteri(target, GL_TEXTURE_BASE_LEVEL, 0);
         glTexParameteri(target, GL_TEXTURE_MAX_LEVEL, static_cast<GLint>(texture.levels() - 1));
 
-        if (gli::gl::EXTERNAL_RED != format.External) // skip for properly greyscale-alpfa fonts textures
+        if (gli::gl::EXTERNAL_RED != format.External) // skip for proper greyscale-alpha font textures
             glTexParameteriv(target, GL_TEXTURE_SWIZZLE_RGBA, &format.Swizzles[gli::SWIZZLE_RED]);
 
         glm::tvec3<GLsizei> const tex_extent(texture.extent());
 
+        GLenum err;
         switch (texture.target())
         {
         case gli::TARGET_2D:
         case gli::TARGET_CUBE:
             CHK_GL(glTexStorage2D(target, static_cast<GLint>(texture.levels()), format.Internal,
                            tex_extent.x, tex_extent.y));
+            if ((err = glGetError()) != GL_NO_ERROR)
```

Comment:
Remove CHK_GL to make this work (CHK_GL uses glGetError())

Reply:
Isn't CHK_GL active only on Debug builds? This is happening on release builds.

Reply:
I believe it's better to have a consistent behavior in all builds.